src/mlpack/methods/linear_svm/linear_svm.hpp

```diff
@@ -119,6 +119,20 @@ class LinearSVM
             const double lambda = 0.0001,
             const double delta = 1.0,
             const bool fitIntercept = false);
+  /**
+   * Initialize the Linear SVM without performing training.  Default
+   * value of lambda is 0.0001.  Be sure to use Train() before calling
+   * Classify() or ComputeAccuracy(), otherwise the results may be meaningless.
+   *
+   * @param numClasses Number of classes for classification.
+   * @param lambda L2-regularization constant.
+   * @paran delta Margin of difference between correct class and other classes.
+   * @param fitIntercept add intercept term or not.
+   */
+  LinearSVM(const size_t numClasses = 0,
```

Comment:
Instead of saying `be sure to use Train() before calling Classify() or ComputeAccuracy()`, it seems to me like we could just check in those functions whether or not the model has 0 classes, and issue an error if so.

Reply:
I have added this part. Please take a look.
Sorry for the delay.